tracer/src/Datadog.Trace/Ci/Tags/TestTags.cs

```diff
@@ -139,5 +139,30 @@ internal static class TestTags
         /// Test command working directory
         /// </summary>
         public const string CommandWorkingDirectory = "test.working_directory";
+
+        /// <summary>
+        /// Test skipped by intelligent test runner
+        /// </summary>
+        public const string SkippedByIntelligentTestRunner = "test.skipped_by_itr";
+
+        /// <summary>
+        /// Test skipped reason by Intelligent test runner
+        /// </summary>
+        public const string SkippedByIntelligentTestRunnerReason = "Skipped by Datadog's Intelligent Test Runner";
```

Comment:
It'd be nice to have some consistency in the skip reason across different tracers, I think. In Java I use `Skipped by Datadog Intelligent Test Runner`, and in Python it's something entirely different :(

Reply:
Yes, I agree with you, I'll change it to use the same as Java, for python let's tell them in the guild meeting.

Reply:
you can leave this if you feel more comfortable, but the UI will handle this message if it detects that `test.skipped_by_itr` is there. If we leave `test.skip_reason` empty we can use it in the future for things like "skipped because we found code coverage for this test in commit <SHA> and there were no changes between <CURRENT_SHA> and <SHA>"

Reply:
done in [89bd903](https://github.com/DataDog/dd-trace-dotnet/pull/4458/commits/89bd90348a7209172cdad0c055df8d9797f2abf0)

Reply:
I prefer to handle the backend skip reason later and not introduce any kind of support in this PR.

Reply:
do you mean you will set `test.skip_reason` to `skipped by datadog's ITR` or not? https://github.com/DataDog/dd-trace-dotnet/pull/4458/commits/89bd90348a7209172cdad0c055df8d9797f2abf0#diff-6f2b3fc2e302b6c710747bcc9ea241c369584fef14164af3ee5946468332108cR31

Reply:
yes, I think it's the right thing to do for now, we still don't have the UI support and we can work later on the custom skip reason.